frontend/apps/marketing/src/types/contentful/entries/Link.ts

```diff
@@ -0,0 +1,10 @@
+import {EntrySkeletonType} from 'contentful';
+
+export type Link = {
+  ariaLabel?: string;
+  isThisAnExternalLink: boolean;
+  label: string;
+  primaryTarget: string;
+};
+
+export type LinkEntry = EntrySkeletonType<Link>;
```

Comment:
As mentioned above, if we do type this way, Entries won't have metadata and sys attributes which we might need in some places. I think we should keep ContentfulEntry generic type and continue using it.

Reply:
I missed the Contentful `Entry` type which adds this functionality. Unfortunately this type doesn't work in our context because we're not using dynamic components at this time. However, I have added an equivalent type, `Entry` which enhances `BaseEntry` into `EntrySkeletonType`.